src/coreclr/src/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -90,7 +90,6 @@ The .NET Foundation licenses this file to you under the MIT license.
 
   <ItemGroup Condition="$(IlcSystemModule) == ''">
     <AutoInitializedAssemblies Include="System.Private.CoreLib" />
-    <AutoInitializedAssemblies Include="System.Private.DeveloperExperience.Console" Condition="$(IlcDisableUnhandledExceptionExperience) != 'true'"  />
```

Comment:
Had to reorder this because we now depend on reflection being initialized (the dictionary used by AppContext grabs the default comparer for `string`)...